uiowa/uiowa
# VWU migration part deux

Resolves #3103 

Migration addition for `blog` content type from D7 site, following the `blog_entry_image_large` migration.





# How to test

Primarily needs a quick check that no errant changes made it in that shouldn't be (composer updates, breaking stuff, stuff unrelated to blog entry migration, etc.).

Need a local copy of writinguniversity.org in Dev Desktop.
Pull the `vwu` multisite from the Graduate College Acquia Cloud docroot.
Leave the DD Apache/MySQL stack running.

`git checkout vmu-blog-migration` and `git pull`
`blt ds --site=writinguniversity.org`
`drush @org-writinguniversity.local en writinguniversity_migrate`
Need to rollback the entries: `drush @org-writinguniversity.local mr d7_writinguniversity_blog_no_image`
`drush @org-writinguniversity.local mim d7_writinguniversity_blog_no_image`


Check that migrations run with no errors other than possible image-related `[error]  Unknown format: 0x0 ` error, and that old blog entries have been migrated.

(http://org-writinguniversity.local.drupal.uiowa.edu/news-archive/2011/11/iowa-and-invisible-man-making-blackness-visible, from https://www.writinguniversity.org/blog-entry/iowa-and-invisible-man-making-blackness-visible, for instance.) Note: only keeping title and body copy; no images.
